src/orders/views/OrderDetails/OrderNormalDetails/index.tsx

```diff
@@ -0,0 +1,277 @@
+import { WindowTitle } from "@saleor/components/WindowTitle";
```

Comment:
Would rather have this file named simply "OrderDetails", I think next to OrderDraftDetails it'll be explanatory enough

Reply:
Cannot be OrderDetails because of name collision